config/config.exs

```diff
@@ -63,6 +58,16 @@ config :hex_web, HexWeb.Endpoint,
   pubsub: [name: HexWeb.PubSub,
            adapter: Phoenix.PubSub.PG2]
 
+config :hex_web, HexWeb.Mailer,
+  adapter: Bamboo.SMTPAdapter,
+  server: System.get_env("HEX_SES_ENDPOINT") || "email-smtp.us-west-2.amazonaws.com",
+  port: System.get_env("HEX_SES_PORT") || "587",
+  username: System.get_env("HEX_SES_USERNAME"),
+  password: System.get_env("HEX_SES_PASSWORD"),
+  tls: :if_available, # can be `:always` or `:never`
+  ssl: false,
```

Comment:
What's the difference between `:tls` and `:ssl`? I think we would want `:always` and `true` for these options?